python-client/tests/scan/test_performance_bias_detector.py

```diff
@@ -32,7 +32,8 @@ def test_performance_bias_detector_trims_large_dataset(german_credit_model, germ
         detector.run(german_credit_model, large_dataset)
     except ValueError:
         pass
-
+    except TypeError:
+        pass
```

Comment:
Why are they passed here? Are there some nuisances between pydantic 1&2 on this?

Reply:
So as I understood the test, the detector will fail anyway, the important stuff is that slice is called, et sample also.
In this case, a new error type was raised, so i'm ignoring it also.

I guess I could have changed exception type to Exception overall